### PR TITLE
[DOCS] typo in date_histogram aggregation example

### PR DESCRIPTION
**Pull request by immon**
The field name fix

### CODE REVIEW

File: docs/reference/aggregations/bucket/datehistogram-aggregation.asciidoc [hunk 1]
```diff
@@ -705,7 +705,7 @@ POST /sales/_search?size=0
 --------------------------------------------------
 // TEST[setup:sales]
 
-<1> Documents without a value in the `publish_date` field will fall into the
+<1> Documents without a value in the `sale_date` field will fall into the
```
Comment (abdonpijpelink) [inline docs/reference/aggregations/bucket/datehistogram-aggregation.asciidoc:708]:
```suggestion
<1> Documents without a value in the `date` field will fall into the
```

**Review by abdonpijpelink — COMMENTED** [review]
I've left a suggestion. Shouldn't this be `date` field?

**Comment by immon** [comment]
> I've left a suggestion. Shouldn't this be `date` field?

Indeed, it should be `date`

**Review by abdonpijpelink — APPROVED** [review]
LGTM. Thanks @immon !

**Pull request merged**